root_files/projects/calendar/caldata

```diff
@@ -0,0 +1 @@
+../../../static/caldata
```

Comment:
This is a symlink to the caldata files. This used to be handled by a custom whitenoise subclass, but this is much more straight-forward and less code. I'd have just moved the files but they need to be served by both URLs.